cli/src/commands/fix.rs

```diff
@@ -484,9 +493,18 @@ fn get_tools_config(ui: &mut Ui, settings: &UserSettings) -> Result<ToolsConfig,
         .table_keys("fix.tools")
         // Sort keys early so errors are deterministic.
         .sorted()
-        .map(|name| -> Result<ToolConfig, CommandError> {
+        .filter_map(
+            |name| -> Option<Result<(RawToolConfig, &str), CommandError>> {
+                match settings.get::<RawToolConfig>(["fix", "tools", name]) {
+                    Ok(tool) if tool.enabled => Some(Ok((tool, name))),
+                    Ok(_) => None,
+                    Err(err) => Some(Err(err.into())),
```

Comment:
nit: type annotation can be omitted if `err` wasn't converted here

That said, it might be better to parse tools including disabled ones. The error message for `tools_config.tools.is_empty()` would be a bit confusing if all tools are disabled.